spring-boot/src/main/resources/hotrod-client.properties

```diff
@@ -0,0 +1 @@
+infinispan.client.hotrod.server_list=127.0.0.1:11222
```

Comment:
In general I think that's a good idea... However we use [a lot of those properties](https://github.com/infinispan/infinispan/blob/master/client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/ConfigurationProperties.java) and duplicating them in the starter adds some additional maintenance effort.

I've created [an issue for implementing this](https://github.com/infinispan/infinispan-spring-boot/issues/8) but I will probably look into it after the initial release.